src/dbcp/transform/gridstatus.py

```diff
@@ -541,24 +639,16 @@ def _transform_pjm(iso_df: pd.DataFrame) -> pd.DataFrame:
     iso_df.drop(columns="point_of_interconnection", inplace=True)
     iso_df.rename(columns={"project_name": "point_of_interconnection"}, inplace=True)
 
-    # GridStatus also wrongly sources "Proposed Completion Date" from
-    # "Revised In Service Date". It should come from "Commercial Operation Milestone".
-    # Switch the column names.
-    iso_df.rename(
-        columns={"proposed_completion_date": "Revised In Service Date"}, inplace=True
-    )
-    iso_df.rename(
-        columns={"Commercial Operation Milestone": "proposed_completion_date"},
```

Comment:
"Commercial Operation Milestone" is in the raw data, but GS does not include it.